src/JBrowse/Store/SeqFeature/BigWig.js

```diff
@@ -97,7 +97,7 @@ return declare([ SeqFeatureStore, DeferredFeaturesMixin, DeferredStatsMixin ],
     },
 
     _load: function() {
-        this._read( 0, 512, lang.hitch( this, function( bytes ) {
+        this._read( 0, 2000, lang.hitch( this, function( bytes ) {
```

Comment:
should be 2048?  Not sure it would make any difference.

Reply:
The bigbed file I tested had a larger header than 512 so I arbitrarily increased it a little. There might be a better value to use but it doesn't hurt unless maybe there was a empty bigwig/bigbed file less than 2000bytes in length